corehq/apps/domain/templates/login_and_password/domain_mobile_reset_pwd_email.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this email because you requested a password reset for your user account at {{ site_name }}{% endblocktrans %}.
+
+{% trans "Please go to the following page and choose a new password:" %}
```

Comment:
Currently, despite the misleading filename, this template serves as both the HTML and plaintext template, so making this a link would break the plaintext render. If we care about plaintext emails (Do we still care about plaintext emails? Is that still a thing in 2017?) then we should split this out into two templates, with a link in the HTML one, and this URL in the plaintext one. 

(There is just something about plaintext, isn't there? Plaintext is the vinyl of email. Plaintext smells like a new book.)